backend/README.md

```diff
@@ -10,29 +10,36 @@ In the project directory, you can run the following commands.
 
 To get to the project directory, login to the docker conatainer with the following command: `docker exec -it container_id /bin/bash` where `container_id` is the id of the docker backend container. This id can be found by running `docker ps`.
 
-### Tests:
+## Tests
 
 ### `./manage.py test`
 
 Launches the test runner in the interactive watch mode.<br>
 See the section about [Running tests](https://docs.djangoproject.com/en/2.1/topics/testing/overview/) for more information.
 
-### Migrations:
+## Migrations
 
-### `python manage.py makemigrations backend`
+Django manages the PostgreSQL database through migrations. Whenever a model is changed, a migration is automatically created to update the database tables so that the application and database are always in sync and changes are tracked through version history.
 
-Creates new migrations based on the changes you've made to your models.<br><br>
+We've also used migrations to seed the database with any public sample data, like the test content for the eMIB sample test.
 
-### `python manage.py migrate backend`
+See [Django Migrations](https://docs.djangoproject.com/en/2.1/topics/migrations/) for more information.
 
-Applies migrations that haven't been migrated yet.<br><br>
+### Scripts
 
-### `python manage.py showmigrations backend`
+You'll have to login to the backend container to run these scripts: `docker exec -it project-thundercat_backend_1 /bin/bash`
 
-Lists project's migrations and their status.<br><br>
+Create new migrations based on the changes you've made to your models:
+`python manage.py makemigrations backend`
 
-### `python manage.py migrate <migration_name> backend`
+Apply migrations that haven't been migrated yet:
+`python manage.py migrate backend`
```

Comment:
Here, we need to do:
`python manage.py migrate` OR `python manage.py migrate custom_models` if we want to migrate only what's under _custom_models_.